packages/api-server/src/cliHandlers.ts

```diff
@@ -101,13 +101,5 @@ export const bothServerHandler = async (options: BothServerArgs) => {
   })
 }
 
-function coerceRootPath(path: string) {
```

Comment:
We already import from `@redwoodjs/fastify-web`, so might as well import the coerce function too, and get rid of this duplicated implementation